src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -465,11 +465,18 @@ private int CheckAndAdjustPositionForSpeculativeAttribute(int position)
 
         #endregion Helpers for speculative binding
 
-        protected override IOperation GetOperationCore(SyntaxNode node, CancellationToken cancellationToken)
+        protected override IOperation GetOperationCore(IMethodSymbol method, CancellationToken cancellationToken)
         {
-            var csnode = (CSharpSyntaxNode)node;
-            CheckSyntaxNode(csnode);
-            return this.GetOperationWorker(csnode, GetOperationOptions.Lowest, cancellationToken);
+            var csmethod = (MethodSymbol)method;
+            CheckSymbol(csmethod);
```

Comment:
>CheckSymbol(csmethod); [](start = 12, length = 22)

It looks like this helper is using "symbol" as parameter name, I would expect "method" to be used for this call site instead.